### PR TITLE
Update UG for edit and sort function

### PR DESCRIPTION
**Pull request by syamfarh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -115,9 +115,9 @@ Format: `list`
 
 ### Editing a person : `edit`
 
-Edits an existing client's parameter in FAPro.
+Edit a parameter in existing contacts, such as name, address, phone number, email, occupation, appointment date and tags.
```
Comment (Daphne789) [inline docs/UserGuide.md]: Just a minor issue. Could you add a comma before "and"? 
Thank you.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -115,9 +115,9 @@ Format: `list`
...
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [o/OCCUPATION] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [o/OCCUPATION] [a/ADDRESS] [appt/APPOINTMENTDATE] [t/TAG]…​`
```
Comment (Daphne789) [inline docs/UserGuide.md]: Maybe can change the "Format:" into "Precise command format:" as we all have agreed.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -224,6 +232,32 @@ Precise expected output on success:
 Precise expected outputs on failure:
 * Error message shown to the user: "The person already exists in the address book"
 
+### Sorting contacts : `sort`
+
+Sort contact lists by parameter name or appointment date.
+
+Format: `undo PARAMETER`
```
Comment (Daphne789) [inline docs/UserGuide.md]: Maybe can change the "Format:" into "Precise command format:" as we all have agreed

Comment (Daphne789) [reply docs/UserGuide.md]: Shouldn't it be "sort Parameter"?

File: docs/UserGuide.md (hunk 1 above)
Comment (syamfarh) [reply docs/UserGuide.md]: done!

File: docs/UserGuide.md (hunk 2 above)
Comment (syamfarh) [reply docs/UserGuide.md]: done!

File: docs/UserGuide.md (hunk 3 above)
Comment (syamfarh) [reply docs/UserGuide.md]: done!

**Pull request merged**